doozer/doozerlib/cli/scan_osh.py

```diff
@@ -314,6 +314,14 @@ def create_update_ocpbugs_ticket(self, packages: dict):
 
                     self.runtime.logger.info(f"Linked {previous_ticket_id.key} to {issue.key}")
 
+                # Report the latest and previous scan on the ticket as a comment
+                comment = f"This ticket was raised because ART automation found new issues in the " \
```

Comment:
any reason why this goes as a comment and not in the ticket description?

Reply:
I didn't want to pollute the ticket with too much info. But do you think its better to keep in the description?

Reply:
Let's try it out like this